katas/content/superdense_coding/Common.qs

```diff
@@ -0,0 +1,67 @@
+namespace Kata.Verification {
+    open Kata;
```

Comment:
Oddly enough, removing this gives me : `main.qs@(1,1): type error: type ProtocolMessage cannot be converted into a string`

Reply:
Makes sense, the namespaces right now are not hierarchical, so Kata.Verification doesn't have access to Kata namespace. (That's why we call solutions with Kata. prefix)